decentraland/unity-explorer
# fix: Map Displayed as Square Instead of Round

## What does this PR change?
Fix #1452 

All the uses of SoftMask were broken:
- The minimap
- Cards in the Backpack
- Etc.

![Image](https://github.com/user-attachments/assets/351df6cb-2b21-41ed-8df5-671664204387)

![Image](https://github.com/user-attachments/assets/cc8d7b77-2b5e-40cd-93e1-0d94c2d01d6f)

## How to test the changes?
1. Launch the explorer
2. Observe that the shape of the map is round instead of a square.
3. Observe all the backpack's cards are showed correctly.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
